### ISSUE TITLE
Add FromRequestUnmarshaller in akka-http module

### ISSUE THREAD

**Issue by gregorath** [Tinkoff/phobos#220]
Hello!
It seems  that in order to have an Akka-http endpoint that accepts a request with xml in the body and decodes it we need a `FromRequestUnmarshaller` available, as it is expected from `entity` and `as` in the example bellow

```
(post & path("request") & entity(as[Request])){ req =>
      complete(OK, req)
}
```

It look like that the implementation can be the same as for the [FromResponseUnmarshaller](https://github.com/Tinkoff/phobos/blob/master/modules/akka-http/src/main/scala/ru/tinkoff/phobos/akka_http/marshalling/text.scala#L16).

Can we add this? I can open a PR if that is okay

**Comment by valentiay** [comment]
Hi!

That makes sense. Feel free to contribute this instance. 

I wonder, if we can replace `FromResponseUnmarshaller` with `FromEntityUnmarshaller`? I think, it'll be a better option. If it does the trick, of course. I haven't touched this code for a while, so now I don't remember why it's only `FromResponseUnmarshaller`, not `FromEntityUnmarshaller`

**Comment by gregorath** [comment]
Using `FromEntityUnmarshaller` makes sense and seems to work, I will see to open a PR.
Thank you for your reply 🙏

**Comment by gregorath** [comment]
PR opened https://github.com/Tinkoff/phobos/pull/221

**Comment by valentiay** [comment]
Merged and published in 0.16.0
